hail_search/queries/mito.py

```diff
@@ -112,6 +118,215 @@ class MitoHailTableQuery(BaseHailTableQuery):
     }
     SORTS[PATHOGENICTY_HGMD_SORT_KEY] = SORTS[PATHOGENICTY_SORT_KEY]
 
+    def _import_and_filter_entries_ht(
+        self, project_guid: str, num_families: int, project_sample_type_data, **kwargs
+    ) -> tuple[hl.Table, hl.Table]:
+        sample_types = set(project_sample_type_data.keys())
+        if len(sample_types) == 1:
+            return super()._import_and_filter_entries_ht(project_guid, num_families, project_sample_type_data, **kwargs)
+
+        self._has_both_sample_types = True
+        entries_hts_map: dict[str, list[tuple[hl.Table, dict]]] = {}
+        for sample_type in sample_types:
+            if num_families == 1:
+                family_guid = list(list(project_sample_type_data.values())[0].keys())[0]
+                ht, sample_data = self._load_prefiltered_family_ht(family_guid, sample_type, project_sample_type_data, **kwargs)
+            else:
+                ht, sample_data = self._load_prefiltered_project_ht(project_guid, sample_type, project_sample_type_data, **kwargs)
+            entries_hts_map[sample_type] = [(ht, sample_data)]
```

Comment:
why make this a list of length 1? If you are only ever considering 1 project here it would be much clearer to not have a list at all

Reply:
I wanted to use the same  function `_filter_entries_ht_both_sample_types` as is used in the code path for multiple project tables, but a cleaner way to do this would be to convert them to lists when I'm calling `_filter_entries_ht_both_sample_types`.